src/common/libflux/msg_handler.c

```diff
@@ -218,8 +218,8 @@ static void call_handler (flux_msg_handler_t *mh, const flux_msg_t *msg)
         return;
     if (!(rolemask & mh->rolemask)) {
```

Comment:
commit message: "modnernize"

There are a couple more of these to fix in `flux_msg_handler_destroy()`.

Reply:
That whitespace is fixed by later commits. I could move the cleanup commit to the end if that helps?

Reply:
Oh, yeah sorry. Thinko -- I thought I had fixed it in those commits due to a conflict